runtime-sdk/modules/evm/src/precompile/auxiliary.rs

```diff
@@ -0,0 +1,44 @@
+//! Implements additional precompiles that are not part of the EVM specification.
+use evm::{
+    executor::stack::{PrecompileHandle, PrecompileOutput},
+    ExitSucceed,
+};
+use ripemd160::Digest as _;
+use sha2::Sha512Trunc256;
+
+use super::{record_linear_cost, PrecompileResult};
+
+pub(super) fn call_sha512_256(handle: &mut impl PrecompileHandle) -> PrecompileResult {
+    record_linear_cost(handle, handle.input().len() as u64, 60, 12)?; // TODO: sensible gas costs, not sure what to set.
```

Comment:
Probably should be benchmarked against SHA256 and set based on that?

Reply:
Ok, did some benchmarks. Seems like the base cost is a bit higher, and then it grows similarly to sha256. I updated the values and pushed the benchmarks. There's also a doc comment pointing to the benchmarks, for anyone revisiting this in future.

![comp](https://github.com/oasisprotocol/oasis-sdk/assets/3691472/045ba93d-a625-4007-8d32-5d8ef7917d0c)